app/gui/view/graph-editor/src/builtin/visualization/java_script/table.js

```diff
@@ -11,7 +11,7 @@ loadStyleFromString(scrollbarStyle)
 // ===========================
 
 class TableVisualization extends Visualization {
-    static inputType = 'Any'
+    static inputType = 'Standard.Table.Data.Table.Table'
```

Comment:
> Unit tests have been written where possible.

Is there a test to fail if the `Table` type is renamed? Types get renamed quite frequently in the stdlib and I'd be sad to see this functionality stop working without a warning/failing test.

Reply:
> > Unit tests have been written where possible.
> 
> Is there a test to fail if the `Table` type is renamed? Types get renamed quite frequently in the stdlib and I'd be sad to see this functionality stop working without a warning/failing test.

That is a good point. But I am somewhat unsure how to do this, as we don't currently have working integration tests.
Do you think it would be sensible to put in a test in the standard library that checks the name of the type of `Table` and says something like "If you need to change this test, also update `visualization/java_script/table.js:14`? "